packaging/chocolatey/crc/tools/chocolateyuninstall.ps1

```diff
@@ -1,8 +1,6 @@
-﻿$ErrorActionPreference = 'SilentlyContinue';
+﻿$ErrorActionPreference = 'Stop';
 $toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 $setupScript = Join-Path "$toolsDir" "crcprerequisitesetup.ps1"

```

Comment:
@anjannath so here `ErrorActionPreference = 'Stop'` but after that we have som command does those command make use of it because `Remove-LocalGroup -Name 'crc-users' -Confirm:$false -ErrorAction SilentlyContinue` is using `SilentlyContinue` as error action.

Reply:
its still going to ignore error for this particular command `Remove-LocalGroup` since we want to ignore the error to handle the case when the group is already deleted